neural_speed/__init__.py

```diff
@@ -434,6 +441,7 @@ def __call__(self, model_input, reinit=False, logits_all=False, **kwargs):
             logits_seq_len_dim = model_input.shape[1] if logits_all else 1
             self.reinit_from_bin = False
             self._check_max_request_num(batch_size, **kwargs)
+            self._check_ctx_size(kwargs.get("ctx_size", 0), multi_round=(not reinit))
```

Comment:
multi_round=(not reinit)   ?

should be bool right?

Reply:
it is bool